### ISSUE TITLE
git annex enableremote google mute-api-lockdown-warning=true broken

### ISSUE THREAD

**Issue by liljenstolpe** [Lykos153/git-annex-remote-googledrive#45]
I am getting the 'lockdown warning' when I do commands like
```bash
$ git annex whereis
```

In that warning, is the instruction to do 
```bash
$ git annex enableremote <remotename> mute-api-lockdown-warning=true
```
However, when I do that command, the remote gets enabled, but the mute-api-lockdown-warning parameter is unexpected.  See the results below 

```bash
$ git annex enableremote my_remote_name mute-api-lockdown-warning=true
enableremote my_remote_name 
git-annex: Unexpected parameters: mute-api-lockdown-warning
failed
git-annex: enableremote: 1 failed
```

**Comment by Lykos153** [comment]
Thanks for the report! Fixed in v1.1.1. Also, the warning message is now automatically muted if you use your own credentials.

**Comment by liljenstolpe** [comment]
Thx for the quick turn around!
